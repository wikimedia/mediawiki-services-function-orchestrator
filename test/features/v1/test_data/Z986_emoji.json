{
	"Z1K1": {
		"Z1K1": "Z9",
		"Z9K1": "Z7"
	},
	"Z7K1": {
		"Z1K1": {
			"Z1K1": "Z9",
			"Z9K1": "Z8"
		},
		"Z8K1": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z17"
				},
				"Z17K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z881",
					"Z881K1": "Z86"
				},
				"Z17K2": {
					"Z1K1": "Z6",
					"Z6K1": "Z886K1"
				},
				"Z17K3": {
					"Z1K1": {
						"Z1K1": "Z9",
						"Z9K1": "Z12"
					},
					"Z12K1": []
				}
			}
		],
		"Z8K2": {
			"Z1K1": "Z9",
			"Z9K1": "Z1"
		},
		"Z8K3": [],
		"Z8K4": [
			{
				"Z1K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z14"
				},
				"Z14K1": {
					"Z1K1": "Z9",
					"Z9K1": "Z886"
				},
				"Z14K4": {
					"Z1K1": "Z6",
					"Z6K1": "Z986"
				}
			}
		],
		"Z8K5": {
			"Z1K1": "Z9",
			"Z9K1": "Z886"
		}
	},
	"Z886K1": [
		{
			"Z1K1": "Z86",
			"Z86K1": {
				"Z1K1": "Z6",
				"Z6K1": "\ud83d\udc69"
			}
		},
		{
			"Z1K1": "Z86",
			"Z86K1": {
				"Z1K1": "Z6",
				"Z6K1": "\ud83c\udfff"
			}
		},
		{
			"Z1K1": "Z86",
			"Z86K1": {
				"Z1K1": "Z6",
				"Z6K1": "\u200d"
			}
		}
	]
}
