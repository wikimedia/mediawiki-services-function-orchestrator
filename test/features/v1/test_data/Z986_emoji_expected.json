{
	"Z1K1": "Z6",
	"Z6K1": "👩🏿‍"
}
